addons/mail/wizard/mail_resend_message.py

```diff
@@ -81,20 +84,38 @@ def cancel_mail_action(self):
         return {'type': 'ir.actions.act_window_close'}
 
 
-class MailResendPartner(models.TransientModel):
-    _name = 'mail.resend.partner'
+class MailResendRecipient(models.TransientModel):
+    _name = 'mail.resend.recipient'
     _description = 'Partner with additional information for mail resend'
 
     notification_id = fields.Many2one('mail.notification', string='Notification', required=True, ondelete='cascade')
     partner_id = fields.Many2one('res.partner', string='Partner', related='notification_id.res_partner_id')
-    name = fields.Char(related='partner_id.name', string='Recipient Name', related_sudo=False, readonly=False)
-    email = fields.Char(related='partner_id.email', string='Email Address', related_sudo=False, readonly=False)
+    name = fields.Char(compute='_compute_details', inverse='_inverse_details', string='Recipient Name', compute_sudo=False, readonly=False, store=True)
```

Comment:
I see `compute_sudo=False`, is there a test that check that we can not read the name / email if we can't read the partner ?